docs/INSTALL.md

```diff
@@ -42,8 +42,7 @@ cd mmdetection
 d. Install mmdetection (other dependencies will be installed automatically).
 
 ```shell
-pip install mmcv
-python setup.py develop  # or "pip install -v -e ."
+pip install -v -e .[all]
```

Comment:
For most users, minimal installation should be enough. Maybe we can just set the default command to `pip install -v -e .`?

Reply:
Sounds reasonable. Changed.